### ISSUE TITLE
What is a Minecraft Research project?

### ISSUE THREAD

**Issue by jmadler** [minecrafthome/minecrafthome#61]

**Comment by ChipWolf** [comment]
Is this a serious question @jmadler?

**Comment by jmadler** [comment]
Yes!  Sorry for my terseness.  I saw the project listed in the BOINC setup flow and am unclear as to what kind of research this computing would support.  Neither the website nor this source code appear to give any examples of what the computations may be.

Specifically I'm looking to support ongoing research in the topic of simulation engines and classifications of their capabilities, relationship between implementation complexity and product complexity, etc.

**Comment by AcaiBerii** [comment]
> Yes! Sorry for my terseness. I saw the project listed in the BOINC setup flow and am unclear as to what kind of research this computing would support. Neither the website nor this source code appear to give any examples of what the computations may be.
> 
> Specifically I'm looking to support ongoing research in the topic of simulation engines and classifications of their capabilities, relationship between implementation complexity and product complexity, etc.

Think BitCoin mining. More specifically, I came here to help crack Minecraft "seeds" and find more answers to some of Minecraft's unsolved questions. Maybe, title screen seeds, PewDiePie's Minecraft seed, really anything that's related to Minecraft that hasn't ever been solved. I'm assuming to crack seeds, it runs through all areas of a Minecraft world and tries to see if a part of a world can be found that relates to a part of a world that we want.

Otherwise, I'm not exactly sure what it does. I'm extremely sorry for the late response.
